components/engine_rocks/src/engine.rs

```diff
@@ -66,6 +66,10 @@ impl RocksEngine {
         self.shared_block_cache = enable;
     }
 
+    pub fn get_shared_block_cache(&self) -> bool {
```

Comment:
Should be `shared_block_cache`. Note we will remove the field in the future and cache will be always shared.